k8s/apis/federation/v1/catalog_types.go

```diff
@@ -35,6 +37,26 @@ type CatalogList struct {
 	Items           []Catalog `json:"items"`
 }
 
+// +kubebuilder:object:root=true
+// +kubebuilder:subresource:status
```

Comment:
unnecessary subresource status and root declaration

Reply:
fixed, thanks for reminding

Reply:
Looks like root=true is needed. Sorry for confusion.